part-spec/common/powerSequence.json

```diff
@@ -0,0 +1,128 @@
+                    "description": "Signal that comes up second in the sequence ",
+                    "examples": [
+                        "vdd",
+                        "en"
+                    ],
+                    "type": "string"
+                },
+                "signal2TerminalIdentifiers": {
+                    "description": "List of component pins associated with signal 2",
+                    "type": "array",
+                    "examples": [
+                        "23",
+                        "A1"
+                    ],
+                    "items": {
+                        "$ref": "string"
+                    }
+                },
+                "timeCondition": {
+                    "description": "Time between the the signal 1 and signal 2 event",
+                    "comment": "units of time",
+                    "$ref": "../common/values.json#/valueOptions"
+                },
+                "powerDirection": {
+                    "description": "Whether this is a power up or power down event",
+                    "enum": [
+                        "Up",
+                        "Down"
+                    ]
+                },
+                "nextPowerState": {
+                    "description": "Power state the device enters after the signal transition",
+                    "type": "string",
+                    "examples": [
+                        "suspend",
+                        "active"
+                    ]
+                },
+                "currentPowerState": {
+                    "description": "Power state the device is currently in before the signal transition",
+                    "type": "string",
+                    "examples": [
+                        "suspend",
+                        "active"
+                    ]
+                },
+                "riseTime": {
+                    "description": "Time for a signal to go from low to high",
+                    "comment": "units of time",
+                    "$ref": "../common/values.json#/valueOptions"
+                },
+                "fallTime": {
+                    "description": "Time for a signal to go from high to low",
+                    "comment": "units of time",
+                    "$ref": "../common/values.json#/valueOptions"
+                },
+                "transitionStartCondition": {
+                    "description": "The percentage of max voltage where rise or fall time starts being measured for timing",
+                    "examples": [
+                        "10%",
+                        "90%"
+                    ],
+                    "comment": "units of percentage",
+                    "$ref": "../common/values.json#/valueOptions"
+                },
+                "transitionEndCondition": {
+                    "description": "The percentage of max voltage where rise or fall time stops being measured for timing",
+                    "examples": [
```

Comment:
If we're assuming that signal2 ends the sequencing, do we want to add that in the description?